### ISSUE TITLE
Raise new issues corresponding to Play Store review criticism from SealNote legacy

### ISSUE THREAD

**Issue by ajburley** [hawknetwork/sealnote-plus#38]
Raise new issues on GitHub corresponding to Play Store review comments from SealNote legacy that are rated (by Play Store) as "Critical". Go through these and process them:

- Close as appropriate (e.g. won't fix, obsolete, already implemented, duplicate etc)
- Mark up with suitable labels and provide explanations
- Assign to milestone(s) as appropriate
- Where multiple items are raised in a single issue, create multiple additional issues and then close the original one

This will enable us to continue working on those items that are still relevant.

**Comment by ajburley** [comment]
Raised #40. The only other reported issue is the Android 8+ backup issue which we will raise when importing from the legacy project issue log, but we still need to add the Play Store comments there.

**Comment by ajburley** [comment]
Blocked by #29
